src/containerapp/azext_containerapp/tests/latest/test_containerappjob_preview_scenario.py

```diff
@@ -4,67 +4,36 @@
 # --------------------------------------------------------------------------------------------
 
 import os
-import time
-from time import sleep
 from azure.cli.testsdk import (ScenarioTest, ResourceGroupPreparer, JMESPathCheck, live_only)
-from subprocess import run
 
-from .common import (write_test_file, TEST_LOCATION, clean_up_test_file)
+from azure.cli.testsdk.decorators import serial_test
+from .common import TEST_LOCATION
 from .custom_preparers import ConnectedClusterPreparer
-from .utils import create_containerapp_env
+from .utils import create_containerapp_env, create_extension_and_custom_location
 
 TEST_DIR = os.path.abspath(os.path.join(os.path.abspath(__file__), '..'))
 
 
 class ContainerappJobPreviewScenarioTest(ScenarioTest):
-
+    @serial_test()
     @ResourceGroupPreparer(location=TEST_LOCATION, random_name_length=15)
     @ConnectedClusterPreparer(location=TEST_LOCATION)
     def test_containerappjob_preview_environment_type(self, resource_group, infra_cluster, connected_cluster_name):
         self.cmd('configure --defaults location={}'.format(TEST_LOCATION))
```

Comment:
May I ask why need to config the default value of location intead of inputting manually?

Reply:
That is because we need all test resources created in same region, and we have vendor to run tests in all regions one by one.
In core, we have this PR to avoid impact others' services: [{containerapp} add random config directory for test by Greedygre · Pull Request #27382 · Azure/azure-cli (github.com)](https://github.com/Azure/azure-cli/pull/27382).